CRM/Utils/Check.php

```diff
@@ -201,15 +184,11 @@ public function assertValid($messages = NULL, $threshold = \Psr\Log\LogLevel::ER
   public static function checkAll($max = FALSE) {
     $messages = self::checkStatus();
 
-    uasort($messages, [__CLASS__, 'severitySort']);
-
     $maxSeverity = 1;
     foreach ($messages as $message) {
-      if (!$message->isVisible()) {
-        continue;
+      if ($message->isVisible()) {
+        $maxSeverity = max(1, $message->getLevel());
```

Comment:
```suggestion
        $maxSeverity = max($maxSeverity, $message->getLevel());
```
I was banging my head trying to figure out why this wasn't working for me.

Reply:
🤦 OMG, you're right, that made no sense. I just copy-pasted...